mlir/lib/Tools/PDLL/Parser/CMakeLists.txt

```diff
@@ -15,3 +15,5 @@ llvm_add_library(MLIRPDLLParser STATIC
   MLIRSupport
   MLIRTableGen
   )
+
+add_mlir_library_install(MLIRPDLLParser)
```

Comment:
What kind of libraries aren't to be installed? Why is this a separate step than `llvm_add_library` ? Should this be behind a macro we can use everywhere instead of having this separate step?

Reply:
`add_mlir_library()` already adds `install()` commands automatically, but these three specifically use `llvm_add_library()` instead for whatever reason, which lacks this.
Should they be converted to `add_mlir_library()` instead, perhaps?

Reply:
If all the other libs in `mlir/lib` are using `add_mlir_library()` then I'm not sure why these aren't indeed.

Reply:
Maybe we don't want them in libMLIR.so for some reasons?